sabnzbd/filesystem.py

```diff
@@ -1272,15 +1273,37 @@ def directory_is_writable_with_file(mydir, myfilename):
 
 
 def directory_is_writable(test_dir: str) -> bool:
```

Comment:
I want a super basic test for this. 
These should **not** return `False` when run in the testing suite, otherwise the test is wrong.

Reply:
> I want a super basic test for this.

So ... move the unicode and long file testing to a different location ... ?

> These should **not** return `False` when run in the testing suite, otherwise the test is wrong.

So ... return nothing ... ? Return True when run in the testing suite? Not run from testing suite? I don't understand. In the testing suite it should be OK because it's being tested on a modern FS without FAT-like limitations, so I don't understand how the test can be wrong.

Reply:
No, I just want you to add this to a `test_filesystem`. Just a simple call to a test-folder.